src/functions/members.js

```diff
@@ -0,0 +1,49 @@
+import {db} from '../Firebase/FirebaseConfig';
+import { collection, getDocs, doc, getDoc, setDoc, addDoc, deleteDoc } from 'firebase/firestore';
+
+async function doesMemberExist(snapshot) {
+    if (!snapshot.exists()) throw new Error('Member not found.');
+}
+
+// Change 'Users' path to appropriate members path when the collection is created
+async function getMembers() {
+    const membersCollection = collection(db, 'users');
```

Comment:
You can probably move all the collections to a file and export it from there. It is just a set of constants and config. 

users is also a magic string

Reply:
And this is called 'membersCollection' but in line 27 there is actually a members collection. Maybe clean this up to usersCollection